src/App.tsx

```diff
@@ -135,12 +138,8 @@ function App() {
           <Publications />
         </Route>
         <Route path="/:language/Unity">
-          <Explore initialFilters={(() => {
-            let initialFilters = getEmptyFilters();
-            initialFilters.unity_aligned_only = true;
-            return initialFilters;
-          })()}/>
-          <span>{/*Temp hack to force page to rerender if already on explore*/}</span>
+          {/*Key added to differentiate Unity view from Explore view and force a rerender*/}
```

Comment:
> Looks good. Just out of curiosity, could you share the creds for analytics? It would be nice to see them to get a better understanding of what parts of the app could use more analytics events

@brett-stephen I can put 10 mins in your calendar to get you setup and walk you through GA